code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -24,12 +24,20 @@ AI MODULES
 
 /obj/item/aiModule/Initialize(mapload)
 	. = ..()
+	if(mapload && HAS_TRAIT(SSstation, STATION_TRAIT_UNIQUE_AI) && is_station_level(z))
+		var/delete_module = handle_unique_ai()
+		if(delete_module)
+			return INITIALIZE_HINT_QDEL
 	if(laws)
 		desc += "<br>"
 		for(var/datum/ai_law/current in laws.inherent_laws)
 			desc += current.law
 			desc += "<br>"
 
+///what this module should do if it is mapload spawning on a unique AI station trait round.
+/obj/item/aiModule/proc/handle_unique_ai()
```

Comment:
Can you document this better/make it clear what the return type does here?

Reply:
done